internal-services/catalog/iib-add-fbc-fragment-to-index-image-task.yaml

```diff
@@ -83,49 +83,75 @@ spec:
       script: |
         #!/usr/bin/env bash
         #
+        # shellcheck disable=SC2207,SC2128,SC2178 # requires refactoring to fix these errors
+        # shellcheck disable=SC2005 # false positive as it thinks tekton parameters might be commands
```

Comment:
What are the "false positives"? It would be better to only disable the check selectively for those lines if needed. But I wonder if it's these:

```
        "build_tags": $(echo "$(params.buildTags)"),
          "add_arches": $(echo "$(params.addArches)"),
```

It does look like the `echo` is redundant, doesn't it?